### PR TITLE
Draft: CKO Paypal MVP

### PR DESCRIPTION
**Pull request by Fifciu**
Still have to:
- [x] Write readme
- [x] Merge and rebase: https://github.com/DivanteLtd/vue-storefront/pull/4748
- [x] Check for Customer

### CODE REVIEW

File: packages/checkout-com/nuxt/proxyMiddleware.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ const getStoredMethods = async ({ customerId }) => {
     return data;
   } catch (err) {
     console.log(err);
+    if (err.response) {
+      console.log(err.response.data);
```
Comment (Qrzy) [inline packages/checkout-com/nuxt/proxyMiddleware.js:37]: Again, I think it's better to `console.error` errors.

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,16171 @@
+Arguments:
```
Comment (Qrzy) [inline yarn-error.log:1]: Let's get rid of this one. :)

**Review by Qrzy — APPROVED** [review]

File: packages/checkout-com/README.md [hunk 3]
```diff
@@ -164,7 +164,10 @@ const payment = await makePayment({ cartId: cart.value.id, email: user.value &&
 // If you've already loaded available payment methods with same useCko composable instance
 const payment = await makePayment();
 
-if (!payment) return;
+if (error.value) {
+    console.log(error.value);
```
[the remark this answers is not in the document]
Comment (Fifciu) [reply packages/checkout-com/README.md:168]: Also changed to `error.value.message` as error.value is instance of `Error`

File: packages/checkout-com/src/useCko.ts [hunk 4]
```diff
@@ -103,16 +102,11 @@ const useCko = () => {
...
-      };
     } else if (selectedPaymentMethod.value === CkoPaymentType.PAYPAL) {
-      finalizeTransactionFunction = () => {
-        console.log('Making transaction with PayPal...');
-      };
+      finalizeTransactionFunction = makePaypalPayment;
+      localError = paypalError;
     } else {
-      error.value = 'Not supported payment method';
+      error.value = new Error('Not supported payment method');
```
Comment (andrzejewsky) [inline packages/checkout-com/src/useCko.ts:109]: why you are returning a new error instead of string?

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

File: packages/checkout-com/src/useCko.ts (hunk 4 above)
Comment (Fifciu) [reply packages/checkout-com/src/useCko.ts:109]: When we have `try{...}catch(e){error.value=e;}` parts - `e` is instance of `Error`. I wanted to unify errors in whole package

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**